src/os/bluestore/BlueStore.h

```diff
@@ -426,26 +426,37 @@ class BlueStore : public ObjectStore,
     }
   };
 
-//#define CACHE_BLOB_BL  // not sure if this is a win yet or not... :/
+#define CACHE_BLOB_BL
 
   /// in-memory blob metadata and associated cached buffers (if any)
   struct Blob {
     MEMPOOL_CLASS_HELPERS();
 
-    std::atomic_int nref = {0};     ///< reference count
+#ifdef CACHE_BLOB_BL
+    /* 32 bytes seems to be optimal for cache size. Typically L1 cache line
+     * is 64 bytes long, so other hot data should fit as well. On the other
+     * hand the capacity should cover most situations as serialized blob with
+     * single physical extent + sbid is less than 8 byte long while testing
+     * 4 KiB rand-write across 4 MiB  objects (usual RBD segment size).  */
```

Comment:
As I mentioned at different comment - PR#13337 invalidates an assumption about single entry in pextent vector as a regular case...